client/my-sites/marketplace/pages/submission-success/signup-success.tsx

```diff
@@ -0,0 +1,72 @@
+import { ThemeProvider, Global, css } from '@emotion/react';
+import { Button } from '@wordpress/components';
+import { useTranslate } from 'i18n-calypso';
+import theme from 'calypso/my-sites/marketplace/theme';
+import './style.scss';
+import MasterbarStyled from '../../components/masterbar-styled';
+import { WelcomeCartIcon } from './welcome-cart-icon';
+import { WelcomeConfigIcon } from './welcome-config-icon';
+import { WelcomeLogoIcon } from './welcome-logo-icon';
+
+const SignupSuccess = (): JSX.Element => {
+	const translate = useTranslate();
+
+	return (
+		<ThemeProvider theme={ theme }>
+			{ /* Using Global to override Global masterbar height */ }
+			<Global
+				styles={ css`
+					body.is-section-marketplace {
+						--masterbar-height: 72px;
+					}
+				` }
+			/>
+			<MasterbarStyled
+				onClick={ () => ( document.location.href = `${ document.location.origin }` ) }
+				backText={ translate( 'Back to WordPress.com' ) }
+			/>
+			<div className="signup-success">
+				<div className="signup-success__header">
+					<WelcomeLogoIcon />
+					<h1 className="signup-success__header-title wp-brand-font">
+						{ translate( 'We’ ll be in touch' ) }
+					</h1>
+					<p>
+						{ translate(
+							'Thank you for apply to join the WordPress.com marketplace. We’ll be in touch with you very soon to discuss next steps.'
+						) }
+					</p>
+				</div>
+				<div className="signup-success__body">
+					<div className="signup-success__row">
+						<div className="signup-success__row-icon">
+							<WelcomeCartIcon />
```

Comment:
Should use gridicons for these http://automattic.github.io/gridicons/

```suggestion
							<Gridicon icon="cart" />
```
cc @SaxonF